### ISSUE TITLE
Settings Link Missing after 5.0 Upgrade

### ISSUE THREAD

**Issue by ChrisLAS** [Chocobozzz/PeerTube#5459]
### Describe the current behavior

After the upgrade to PeerTube 5.0 I cannot find the PeerTube settings as my root user. The link is missing post-upgrade.

### Steps to reproduce

1. Upgrade a 4.x Install to 5.0
2. Login as Root (or another admin user)
3. No settings link to be found

### Describe the expected behavior

I expect to find a link to the system settings UI.

### Additional information

* PeerTube instance: Jupiter.Tube
  * URL: https://jupiter.tube/
  * Version: 5.0.0
  * NodeJS version:
  * Ffmpeg version:

* Browser name, version and platforms on which you could reproduce the bug: Chrome and Firefox
* Link to browser console log if relevant:
* Link to server log if relevant (`journalctl` or `/var/www/peertube/storage/logs/`):

**Comment by Chocobozzz** [comment]
Hello,

What do you mean by "settings link"?

**Comment by noblepayne** [comment]
Before the upgrade our admin user would see a link for `Administration` in the left-hand side menu under `My library`. After upgrade, this item is no longer presented. We've checked and confirmed that this user is still an admin in the database (role = 0).
![image](https://user-images.githubusercontent.com/33278192/207706581-b43339a6-47be-49f3-a3fd-d39eed323d3d.png)

**Comment by Chocobozzz** [comment]
Can you try to uninstall some plugins?

**Comment by alkahan** [comment]
Same problem here. I have uninstall ed all plugins and it was working again.

I think the problem was caused by one of:
- peertube-plugin-hardware-transcode-vaapi
- peertube-plugin-transcoding-profile-debug

**Comment by nishantsworld** [comment]
Same here. I was able to see the Administration link while installing v4.3, however unable to see it in v5.

Just fyi...
1. It was missing even in v5 RC1, however i thought it might be a miss and will get fixed in the stable build
2. I have not installed any custom plugins, but using the official production-bullseye image
3. Even if i try to manually visit /admin/users/list in the URL (like for v4.3), it simply redirects me to "My Account"

**Comment by gohlool** [comment]
Same here. I though I did something wrong! I first updated from 4.3.1 to 5.0.0 and had problems with the configuration. Downgraded to 4.3.1. Copied the production.yaml from container to host and added the missing configuration back to container. did docker-compose down -v again and docker-compose up -d. Interestingly I had to copy the modified production.yaml back again and did a docker-compose restart. The container started, but after logging in, I couldn't find the "Administration" section.
